edx_django_utils/monitoring/internal/middleware.py

```diff
@@ -462,6 +466,60 @@ def log_corrupt_cookie_headers(self, request, corrupt_cookie_count):
             log.info(piece)
 
 
+class FrontendMonitoringMiddleware:
+    """
+    Middleware for adding the forntend monitoring scripts to the response
+    """
+    def __init__(self, get_response):
+        self.get_response = get_response
+
+    def __call__(self, request):
+        response = self.get_response(request)
+
+        if response.status_code != 200 or not response['Content-Type'].startswith('text/html'):
+            return response
+
+        # .. setting_name: OPENEDX_TELEMETRY_FRONTEND_SCRIPTS
+        # .. setting_default: None
+        # .. setting_description: Scripts to inject to response for frontend monitoring, this can
+        #    have multiple scripts as we support multiple telemetry backends at once, so we can
+        #    provide multiple frontend scripts in a multiline string for multiple platforms tracking.
+        #    Best is to have one at a time for better performance.
```

Comment:
This should be explicit that the value is an HTML string, not e.g. a script URL.